.buildkite/auditbeat/scripts/package-step.sh

```diff
@@ -43,4 +43,18 @@ if are_files_changed "$changeset"; then
           imagePrefix: "${IMAGE_UBUNTU_ARM_64}"
           instanceType: "t4g.large"
 EOF
+else
+    cat <<-EOF
+      steps:
+        - label: ":ubuntu: Packaging Linux X86"
+          key: "package-linux-x86"
+          command:
+            - "buildkite-agent annotate "No required files changed" --style 'warning' --context 'ctx-warning'"
+          notify:
+            - github_commit_status:
+                context: "Auditbeat/Packaging: Linux X86"
+          agents:
+            provider: "gcp"
+            image: "${IMAGE_UBUNTU_X86_64}"
```

Comment:
Use the default agent that runs on K8s, and it's faster to be provisioned and cheaper.

However, I see this part as something that could be improved to avoid using agents for notifying GitHub checks when things are not needed to run. This can mislead since the GitHub check will exist but do nothing,  so users will think the stage actually ran while it did not.

```suggestion
```

Reply:
It will be aligned in future phases.